torch/_inductor/lowering.py

```diff
@@ -3871,6 +3801,28 @@ def sum_(x, axis=None, keepdims=False, *, dtype=None):
         override_return_dtype=torch.bool,
     )
 )
+logical_xor = register_pointwise(
+    aten.logical_xor,
```

Comment:
nit, can be done in follow up, if you send `name=bitwise_xor` arg chances are you won't have to implement logical_xor op, but if it doesn't work, don't bother

Reply:
This works nicely, thanks!
You can actually do same thing for logical_and and logical_or... But maybe that breaks some desired short-circuiting behavior, so I didn't do it.